lib/sky-pages-bootstrap-generator.js

```diff
@@ -4,16 +4,11 @@
 function getBootstrap(skyPagesConfig) {
   let bootstrap = '';
 
-  if (skyPagesConfig.auth || skyPagesConfig.omnibar) {
-    bootstrap =
-`import { SkyAppBootstrapper } from '${skyPagesConfig.runtimeAlias}';
-
-SkyAppBootstrapper.bootstrapConfig = {
-  omnibar: ${JSON.stringify(skyPagesConfig.omnibar)},
-  auth: ${JSON.stringify(skyPagesConfig.auth)}
-};
-
-`;
+  if (skyPagesConfig.packages) {
+    bootstrap = `
+      import { SkyAppBootstrapper } from '${skyPagesConfig.runtimeAlias}';
+      SkyAppBootstrapper.bootstrapConfig = ${JSON.stringify(skyPagesConfig.packages)};
```

Comment:
I can appreciate the efficiency by not having to specify auth, help, and omnibar in two places (here and `runtime/bootstrap.ts`, which has to stay).  

Doing this means we would have to instruct our current consumers to update their `skyuxconfig.json`, which I'm not sure is worth the efficiencies gained.  Generally speaking, we've found being very explicit in what options / configuration we allow has worked best.